fiftyone/core/tags.py

```
@@ -1138,25 +1227,31 @@ def _to_export_doc(doc):
+            anchor=doc.get("anchor", None),
+            created_by=doc.get("created_by", None),
+            last_modified_by=doc.get("last_modified_by", None),
+            created_at=doc.get("created_at", None),
+            last_modified_at=doc.get("last_modified_at", None),
+        )
+    raise ValueError(
+        "Expected a temporal tag document, but found kind=%s" % kind
     )
 
 
 def _build_query(
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_build_query`

(ANN202)